tests/integration/test_ingest_markdown_docs_tool.py

```
@@ -0,0 +1,461 @@
+            assert "chunks_created: 3" in result
+            assert "processing_time: 1.5" in result
+
+            # Verify service was called
+            mock_context.support_doc_service.ingest_markdown_document.assert_called_once()
+
+        finally:
+            os.unlink(tmp_file_path)
+
+    async def test_ingest_directory_batch_processing(self, mock_context, temp_markdown_files):
+        """Test batch processing of a directory with multiple markdown files."""
+        temp_dir, files = temp_markdown_files
```

Pylint (3.3.7)

[warning] Unused variable 'files'

(W0612)